### ISSUE TITLE
Accept GitHub status events from the new TC deployment

### ISSUE THREAD

**Issue by jgraham** [web-platform-tests/wpt.fyi#1605]
TaskCluster is changing instances; we will be on the community instance. This means that the root URL will change to `https://community-tc.services.mozilla.com`. We need to update the wpt.fyi configuration to match; deploying this should presumably be coordinated with actually switching wpt over to the new instance.

For full details see https://bugzilla.mozilla.org/show_bug.cgi?id=1574668

**Comment by foolip** [comment]
@Hexcles are you able to manage this transition this week? If not, @stephenmcgruer?

**Comment by stephenmcgruer** [comment]
@jgraham - is the new root URL already up and running? Assuming so, seems we should just land and deploy a change in wpt.fyi that accepts either URL in `processTaskClusterBuild`, and then migrate WPT.

Looks like the deadline is November 9th from the Mozilla bug.

**Comment by jgraham** [comment]
Yes it's already running. If we can accept both that's perfect (we've taken the same approach with the wpt code itself).

**Comment by djmitche** [comment]
Please ping me if there's anything I can do to help.  From a quick refresher on status, I think I'm out of the critical path now?

**Comment by Hexcles** [comment]
@jgraham suggested to accept both domains. I took a quick look and it's indeed possible if we assume the domain in the GitHub status event from Taskcluster is always the same as the domain of the API server. I created #1610 for this.

And @djmitche , no it's on me and James' coordinating the switchover. We plan to do it tomorrow 3:30pm UTC.

**Comment by djmitche** [comment]
Awesome!  Note, too, that the context in the status event has a different prefix for the two integrations.

**Comment by Hexcles** [comment]
@djmitche Just to confirm, if I use the prefix extracted from the context in a GitHub status as the rootURL to construct API URL using `tcurl.API`, I should be able to query the task group in the context, right?

**Comment by djmitche** [comment]
I think the contexts are strings like "Taskcluster (push)" or "Community-TC (push)", so you'd need to index some array of rootUrls to determine the rootUrl.  But with that modification, yes.

**Comment by djmitche** [comment]
Here's a commit with the new deployment (click the checkmark)

**Comment by Hexcles** [comment]
Ahh *that* context. Sorry I meant the target URL.

And yes, I need to accept the new context string. Thanks for the heads-up!

**Comment by Hexcles** [comment]
Alright, #1610 should do it.

**Comment by Hexcles** [comment]
Reopening this issue to keep track of the rollout and switchover process

**Comment by Hexcles** [comment]
This is now put on hold because of a GitHub incident: https://www.githubstatus.com/incidents/42hkbtl63nmn

We want to make sure GitHub webhook deliveries are normal when we do the switch to avoid potential headache.

**Comment by Hexcles** [comment]
GitHub is recovering, but we are not likely to be able to switch today because of timezones. We'll try again tomorrow.

**Comment by Hexcles** [comment]
For the record, wpt.fyi side is done but we are blocked by https://github.com/web-platform-tests/wpt/issues/20133

**Comment by Hexcles** [comment]
This has been fully fixed. Prod now accepts both old and new Taskcluster URLs. Proof:

This commit runs through both Taskcluster instances:
https://github.com/web-platform-tests/wpt/pull/20175/commits/197d11eef45d50fe70f6618b71b3d95ca07bae29

And wpt.fyi successfully received two pairs:
https://wpt.fyi/runs?sha=197d11eef45d50fe70f6618b71b3d95ca07bae29&max-count=100

We have thus turned off old Taskcluster completely on wpt. Woohoo!

Thanks again for the help, @jgraham & @djmitche !
